jenkins_pipelines/environments/common/pipeline-aws.groovy

```diff
@@ -32,10 +32,10 @@ def run(params) {
         def capybara_timeout =30
         def default_timeout = 300
 
-        if (params.terraform_parallelism) {
-            local_mirror_params = "${local_mirror_params} --parallelism ${params.terraform_parallelism}"
-            aws_mirror_params = "${aws_mirror_params} --parallelism ${params.terraform_parallelism}"
-            env.common_params = "${common_params} --parallelism ${params.terraform_parallelism}"
+        if (params.parallelism) {
```

Comment:
Personally I like to specify that the parallelism here is about concurrent threads during deployment. To distinguish from parallelism during tests. Maybe we can name it `deploy_parallelism` ?